consensus/types/src/light_client_finality_update.rs

```diff
@@ -45,15 +47,9 @@ impl<T: EthSpec> LightClientFinalityUpdate<T> {
         beacon_state: &BeaconState<T>,
         block: &SignedBeaconBlock<T>,
         attested_state: &mut BeaconState<T>,
-        finalized_block: &SignedBlindedBeaconBlock<T>,
+        attested_block: &SignedBeaconBlock<T>,
+        finalized_block: &SignedBeaconBlock<T>,
     ) -> Result<Self, Error> {
```

Comment:
Where is this function used? 
Looks like we now construct the updates from `LightClientServerCache`, so no longer need this?